lib/sidekiq/worker.rb

```diff
@@ -241,6 +257,11 @@ def perform_async(*args)
         client_push("class" => self, "args" => args)
       end
 
+      # Inline execution of job's perform method after passing through Sidekiq.client_middleware and Sidekiq.server_middleware
+      def perform_inline(*args)
+        set({}).perform_inline(*args)
```

Comment:
Is there a way to get rid of this unnecessary `set`?

Reply:
Would moving the logic to `Sidekiq::Client` be an acceptable approach?